### DISCUSSION TITLE
http+docker Error in while installing cQube Starter pack

### DISCUSSION THREAD

**Discussion by lokeshshivamurthy** [Sunbird-cQube/community#871] [Q&A]
Hi Team,
Yesterday while performing the installation of cQube starter pack for a state, the following error came up during installation.

VM Used      :   Ubuntu 22.04 version
Repo cloned  :  https://github.com/Sunbird-cQube/cqube-devops.git
branch used  :  release-v5.0.3

After running install.sh file, following error came up.

![image](https://github.com/user-attachments/assets/7213e276-09aa-40bd-b76f-f1dc24e1c009)

As of now, we can disable the warning.
But error seen :  "Error Connecting: Error while fetching server API Version: Not supported URL scheme http+docker"
We installed latest version of ansible and docker. Even then the issue persists.

Please can you suggest resolution for the error seen.

Attaching the steps of installation for reference.
[Installation Error - Steps.docx](https://github.com/user-attachments/files/16436099/Installation.Error.-.Steps.docx)

**Comment by manjudr** [comment]
@lokeshshivamurthy  - We will check and get back to you as soon as possible.

**Reply by lokeshshivamurthy** [reply]
Thank You !!

**Comment by AshkanPoyber** [comment]
Hi [lokeshshivamurthy](https://github.com/lokeshshivamurthy),

Thank you for sharing the detailed steps and the error details.

The error you are getting:
Error Connecting: Error while fetching server API Version: Not supported URL scheme http+docker


usually appears when Docker is using an unsupported connection scheme or when there is a mismatch between the Docker client and server setup.

Here are a few things you can try:

### 1. Check Docker installation
Make sure Docker is installed and running properly:

```
sudo systemctl status docker
docker version
```

Both the client and server versions should be compatible


2. Verify the DOCKER_HOST variable
If **DOCKER_HOST** is set to something like **http+docker://...,** unset it:
`unset DOCKER_HOST`
Then try running the installation again


3. Fix Docker socket permissions
Make sure your user has proper access to Docker:
```
sudo usermod -aG docker $USER
newgrp docker

```


Then test:
`docker info`


4. Run the installer again
After fixing the environment:
`./install.sh`


5. (Optional) Ensure Docker is using the default context
```
docker context ls
docker context use default
```

If the issue still persists, please share the outputs of:
```
docker info
echo $DOCKER_HOST
```


This will help in identifying the exact cause.

Hope this helps! Let me know if you need more assistance.

**Reply by AshkanPoyber** [reply]
@lokeshshivamurthy  
If the steps I shared helped resolve the issue, please feel free to mark the answer as the accepted solution.  
This will help others facing the same problem. Thanks!
